Classes/Issues/EditTitle/EditIssueTitleViewController.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  EditIssueTitleViewController.swift
+//  Freetime
+//
+//  Created by B_Litwin on 10/22/18.
+//  Copyright © 2018 Ryan Nystrom. All rights reserved.
+//
+
+import UIKit
+import GitHubAPI
+import SnapKit
+
+protocol EditIssueTitleViewControllerDelegate: class {
+    func sendEditTitleRequest(newTitle: String, viewController: EditIssueTitleViewController)
```

Comment:
nit: delegate pattern has `self` param **first**, then other data params. Something like this would be more consistent:

```swift
func sendEditTitleRequest(from viewController: EditIssueTitleViewController, with title: String)
```